src/lib/y2storage/autoinst_issues/could_not_calculate_boot.rb

```diff
@@ -17,12 +17,12 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "y2storage/autoinst_issues/issue"
+require "installation/autoinst_issues/issue"
 
-module Y2Storage
+module Installation
```

Comment:
I think we should not move this code to a different module/namespace. IMHO, it is OK that specific storage-ng issues live under the `Y2Storage::AutoinstIssues` namespace.